R/textplot_network.R

```diff
@@ -44,9 +44,9 @@
 #' @import network ggplot2 ggrepel
 #' @keywords textplot
 textplot_network <- function(x, min_freq = 0.5, omit_isolated = TRUE, 
-                             edge_color = 'skyblue', edge_alpha = 0.5, 
+                             edge_color = '#1F78B4', edge_alpha = 0.5, 
```

Comment:
I like the use of semantic colours better than the hex colours, as defaults. We can use some hex colours in the examples for illustration.

Reply:
The color is actually from a palet (second from the top):
```
 RColorBrewer::brewer.pal(8, 'Paired') 
# [1] "#A6CEE3" "#1F78B4" "#B2DF8A" "#33A02C" "#FB9A99" "#E31A1C" "#FDBF6F" "#FF7F00"
```
 This is an effort to set consistent color scheme across the package's visual output. Unfortunately, the colors do not have names in R and named R colors are not very good (this is why **RColorBrewer** exists).

 